third_party/nvfuser/csrc/ir_internal_nodes.h

```diff
@@ -1877,44 +1877,8 @@ class TORCH_CUDA_CU_API TensorDomain : public Val {
   static bool hasBroadcast(const std::vector<IterDomain*>&);
   static bool hasReduction(const std::vector<IterDomain*>&);
 
-  // Get a vector of the same size as the given rfactor domain filled with true
```

Comment:
nitpick. maybe I missed it, can we also add the description of the new contiguity protocol somewhere in the code?

Reply:
Added in `TensorDomain::contiguity`